### PR TITLE
module 1 completed

### PR DESCRIPTION
**Pull request by Denis-Milashuk**

### CODE REVIEW

File: module_1_task_1/.gitignore
Comment (unix-follower) [inline module_1_task_1/.gitignore:1]: It is not necessary to put this file everywhere you already have a .gitignore file in the root of the project.
Optional: delete it.

File: module_1_task_1/.mvn/wrapper/maven-wrapper.jar
Comment (unix-follower) [inline module_1_task_1/.mvn/wrapper/maven-wrapper.jar:1]: Don't need to duplicate Maven wrapper.
Optional: delete it.

File: module_1_task_1/.mvn/wrapper/maven-wrapper.properties
Comment (unix-follower) [inline module_1_task_1/.mvn/wrapper/maven-wrapper.properties:1]: Don't need to duplicate Maven wrapper.
Optional: delete it.

File: module_1_task_1/mvnw
Comment (unix-follower) [inline module_1_task_1/mvnw:1]: Don't need to duplicate Maven wrapper.
Optional: delete it.

File: module_1_task_1/mvnw.cmd
Comment (unix-follower) [inline module_1_task_1/mvnw.cmd:1]: Don't need to duplicate Maven wrapper.
Optional: delete it.

**Review by unix-follower — APPROVED** [review]

File: module_1_task_1/.gitignore
Comment (Denis-Milashuk) [reply module_1_task_1/.gitignore:1]: fixed

File: module_1_task_1/mvnw.cmd
Comment (Denis-Milashuk) [reply module_1_task_1/mvnw.cmd:1]: fixed

File: module_1_task_1/mvnw
Comment (Denis-Milashuk) [reply module_1_task_1/mvnw:1]: fixed

File: module_1_task_1/.mvn/wrapper/maven-wrapper.jar
Comment (Denis-Milashuk) [reply module_1_task_1/.mvn/wrapper/maven-wrapper.jar:1]: fixed

File: module_1_task_1/.mvn/wrapper/maven-wrapper.properties
Comment (Denis-Milashuk) [reply module_1_task_1/.mvn/wrapper/maven-wrapper.properties:1]: fixed

**Pull request merged**